src/values/WordCombination.js

```diff
@@ -1,180 +1,69 @@
-import { forEach } from "lodash-es";
-import { has } from "lodash-es";
-
-/**
- * Returns whether or not the given word is a function word.
- *
- * @param {string} word The word to check.
- * @param {Function} functionWords The function containing the lists of function words.
- * @returns {boolean} Whether or not the word is a function word.
- */
-function isFunctionWord( word, functionWords ) {
-	return -1 !== functionWords.indexOf( word.toLocaleLowerCase() );
-}
-
 /**
  * Represents a word combination in the context of relevant words.
  *
  * @constructor
  *
- * @param {string[]} words The list of words that this combination consists of.
+ * @param {string} word The word.
+ * @param {string} stem The stem / base form of the word.
```

Comment:
You made this optional. Let's reflect that in the JSDoc.

```suggestion
 * @param {string} [stem] The stem / base form of the word, defaults to the word.
```

Reply:
Right!